src/main/java/nextstep/courses/domain/session/Enrollment.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.CannotEnrollException;
+import nextstep.courses.domain.Amount;
+import nextstep.payments.domain.Payment;
+
+public class Enrollment {
+    private SessionStatus status;
+    private final EnrollmentPolicy enrollmentPolicy;
+    private int enrolledCount = 0;
+
+    public Enrollment(SessionStatus status, EnrollmentPolicy enrollmentPolicy) {
+        this.status = status;
+        this.enrollmentPolicy = enrollmentPolicy;
+    }
+
+    /* ------------ 정책 검증 ------------ */
+    // 모집중 상태인지
+    public boolean isOpen() {
+        return status.equals(SessionStatus.OPEN);
+    }
```

Comment:
```suggestion
    public boolean isOpen() {
        return status.isOpened();
    }
```
- 값을 꺼내서 비교하지 말고 메시지를 보내보세요.